src/plugins/discover/public/application/doc/single_doc_route.tsx

```diff
@@ -85,11 +80,23 @@ const SingleDoc = ({ id }: SingleDocRouteProps) => {
     return <LoadingIndicator />;
   }
 
+  if (!id) {
+    return (
+      <DiscoverError
+        error={
+          new Error(
+            i18n.translate('discover.discoverError.missingIdParamError', {
+              defaultMessage: 'URL query string is missing id.',
```

Comment:
We had almost identical message [here](https://github.com/elastic/kibana/blob/61243fc1cbe660d245a76fc709b6a0e9f0abef14/src/plugins/discover/public/utils/with_query_params.tsx#L27). Upper case `ID` seems doesn't recognised by router (see screenshot below), we should use lower case version or make it not a case sensitive. 

![42F5FBA9-F7B5-41E2-8710-904306DBF44D](https://user-images.githubusercontent.com/39378793/196709472-a009adf2-faae-432a-837d-1465dbf35d0e.jpeg)

Reply:
Oops I missed that we had basically the same message before, and since 'id' is the name of the query parameter, I agree it makes sense to keep.

Reply:
When does this message show up?  It's not clear to me what "Go to Discover main view" means or why the user would want to go there.

Reply:
It's one of those messages that shouldn't show up, you need to mess with the id of the single document view like this:

https://kertal-pr-143117-enhancement-unified-histogram-lens.kbndev.co/app/discover#/doc/ff959d40-b880-11e8-a6d9-e546fe2bba5f/kibana_sample_data_ecommerce

What happens here is that you get the the "main" page of Discover, this is internal wording of the view with the field selector, histogram, data table. Wie could label it maybe

"Go to Discover start view"? 

This would maybe a bit clearer ... open for other wordings